ipaserver/install/server/replicainstall.py

```diff
@@ -1252,11 +1252,9 @@ def promote_check(installer):
             config.kra_host_name = kra_host
             kra_enabled = True
         else:
-            if options.setup_kra:
-                root_logger.error("There is no KRA server in the domain, "
-                                  "can't setup a KRA clone")
-                raise ScriptError(rval=3)
             kra_enabled = False
```

Comment:
This exception is right if you are installing a replica, why did you remove it ?